### DISCUSSION TITLE
Champions Friday August 12, 2022 text standup.

### DISCUSSION THREAD

**Discussion by fire** [omigroup/omigroup#276] [General]
**For Friday's text standup, over the last week from August 5th to August 12th, 2022.**

This week is ACM Siggraph scientific conference for computer graphics locally in Vancouver, Canada.

Met with Adam Morris at Siggraph and talked about GLTF, Godot Engine, and OMI / VRMC.

Adam was interested in asking if a spec writer at Khronos would be able to review the VRM 1.0 beta.

There was talk about how meeting VRMC was difficult because of time zones and how VRM should both part of the Metaverse Forum and as a technical part of the existing 3d format group.

Godot Engine 4 expects to have a beta in few weeks. Will need to merge all of outstanding work in progress. This includes gltf extension import, avatar animation and ik.

**What is iFire doing for the next champions Friday?**

Combine notes from Siggraph 2022.

Assist @mrmetaverse  's project for https://github.com/AngellXR/metaversedev

**What is iFire's blockers?**

None. Probably vacation?

**Comment by mrmetaverse** [comment]
Thanks again, @fire. 

My weekly standup :levitate: 
## This past week: 
- Facilitated the Wednesday meeting. Took notes. 
- Coordinated with Humbletim on some PRs. 
- Created the graphic for the topic. Made announcements, and engaged on Twitter. 
- Community engagement, and as always, encouraging new people to attend the Wednesday meeting. 
- I spoke last minute at a Twitter space yesterday about Metaverse interoperability, and there were a lot of new faces. Maybe some of them will show up in our community. 

## Intentions for this next week: 
- I was grateful to have a theme proposed that challenged the status quo a bit. I like the direction we are headed with demos and look forward to continuing this more-demos approach. I'd like to see some of these discussions turn into more protocol submissions for review/collaboration. 
- The UX Research group: I hope we can vote to form this one, as there are a number of people expressing interest. I think it would unlock new possibilities for our exploration as well. 
- I am happy to facilitate Wednesday again.


## Blockers
I am unsure of how Mee6 works and would like to see us close out the [Milestone: Selecting our Bots](https://github.com/omigroup/omigroup/milestone/3)

## Balance 
This week I would say I feel at a 9 (9/10).

**Comment by RedEagleP1** [comment]
Blockers:  Mee6 is not tagging the welcome team. Try going to the dashboard and slowly typing @welcomer until it gives you the popup and choose it from the popup and that may work

**Comment by antpb** [comment]
## This past week: 
- light meeting/agenda coordination
- participated in gltf extensions meeting where I demoed a new 3D scene editor I'm working on.


## Intentions for this next week: 
- Have a side quest to mimic KHR_audio attributes in a new element in my editor where we will have a way to generate audio assets on the website where folks can download and test against.


## Blockers
My free hours to participate dont always align with when things are happening. Trying to find ways to be here asynchronously more.

## Balance 
Feeling chaotic and clumsy at the moment lol
